RealEstate/1 table.sql

```diff
@@ -0,0 +1,27 @@
+use RealEstate
+GO
+drop table if exists Property
+go
+create table Property( --Not all houses, some are empty lots
+    PropertyId int not null IDENTITY primary key,
```

Comment:
Identity-should not be all caps

Reply:
There is really no difference in SQL server between lower and uppercase, and some developers prefer to use all caps. But I changed it for consistency